templates/cypress/plugins/index.js

```diff
@@ -27,6 +27,7 @@ module.exports = (on, config) => {
 	config.env.WEB3_SYSTEM_TEST_MNEMONIC = process.env.WEB3_SYSTEM_TEST_MNEMONIC;
 	config.env.WEB3_SYSTEM_TEST_PORT = process.env.WEB3_SYSTEM_TEST_PORT;
 	config.env.WEB3_SYSTEM_TEST_PROVIDER = process.env.WEB3_SYSTEM_TEST_PROVIDER;
+	config.env.WEB3_SYSTEM_TEST_CLIENT = process.env.WEB3_SYSTEM_TEST_CLIENT;
```

Comment:
Client is a term that generally refers to the node software being run e.g. Ganache, Geth, etc. I would rename this to `WEB3_SYSTEM_TEST_BROWSER` for clarity

Reply:
but by default, it is `node`
export const DEFAULT_SYSTEM_CLIENT = 'node';
node - is not a browser

Reply:
Then `WEB_SYSTEM_TEST_ENGINE` could work

Reply:
we should refer `client` for eth node, in our repo.

Reply:
I've set env as WEB3_SYSTEM_TEST_ENGINE. thx